### DISCUSSION TITLE
After 6.01, all boards except the Mega 2560 are normal.

### DISCUSSION THREAD

**Discussion by js-midi** [shanteacontrols/OpenDeck#109] [Q&A]
After 6.01, all boards except the Mega 2560 are normal.
But Mega2560 still can't enter open deck configurator
I thought it might be a problem with the board, so I bought another one and uploaded it, but it's the same thing.
  I may be clumsy in some parts, but I hope you will review it one more time.
thank you

**Comment by paradajz** [comment]
I wasn't able to reproduce any such issues on any blank board with v6.0.0. There was, however, a problem with Mega after enabling analog components, but that is solved in v6.0.1. Otherwise, all the boards I have here (which is every supported board actually) work normally. Are both LEDs on Arduino Mega (RX/TX) on in your case?

**Reply by js-midi** [reply]
No, as in 6.0.0, the rx led blinks, but I can't enter the configurator.
If it's my problem, I'll try to solve it
For now, I plan to use only good boards.
thank you

**Reply by paradajz** [reply]
Try flashing mega2560 with the attached firmware.

[mega2560.zip](https://github.com/shanteacontrols/OpenDeck/files/7269267/mega2560.zip)

**Reply by js-midi** [reply]
still the same symptom
The mega2560 is currently in my project.
I would like to exclude
I would like to use only two of them, discovery and stm32f411.
These are the most perfectly working boards I have.
thank you
